LICENSE

```diff
@@ -0,0 +1,31 @@
+Software License Agreement (BSD License)
+
+Copyright (c) 2010, Willow Garage, Inc.
```

Comment:
This copyright seems incorrect. Can you explain?

Reply:
> This copyright seems incorrect. Can you explain?

Oops, copy-n-paste error.  Though I'm not entirely sure what it should be; `2020, Open Source Robotics Foundation, Inc.`?

Reply:
Maybe multiple copyright lines? One for Willow and another for OSRF?

Reply:
The LICENSE file shouldn't have any copyright statements right? It just lists the licenses used, no?

Taking an example of Rust, completely at random, they have dual license and neither has copyright info:

https://github.com/rust-lang/rust/blob/master/LICENSE-APACHE
https://github.com/rust-lang/rust/blob/master/LICENSE-MIT

Reply:
I went with @wjwwood 's suggestion here and removed the copyright line completely from the LICENSE file.